src/mongocxx/options/find.cpp

```diff
@@ -211,6 +214,179 @@ const stdx::optional<class read_preference>& find::read_preference() const {
     return _read_preference;
 }
 
+find& find::convert_all_modifiers() {
+    if (!_modifiers) {
+        return *this;
+    }
+
+    for (auto&& ele : _modifiers->view()) {
```

Comment:
I don't like multi-page functions because they impede readability.  How about breaking up type-specific code to separate functions. E.g.:

```
if (ele.key() == sdx::string_view("$comment")) {
    _convert_comment(ele);
} else if (ele.key() == stdx::string_view("$hint")) {
    _convert_hint(ele);
}...
```